### ISSUE TITLE
Two body integral

### ISSUE THREAD

**Issue by RichRick1** [theochem/ModelHamiltonian#31]
1. We need a function that convert a one and two body integral into anti-symmetrize for two body integral
 @gabrielasd has this function
 @PaulWAyers can help with math

**Comment by PaulWAyers** [comment]
Specifically need to have utilities for:
chemistry <-> physics notation
 <-> antisymmetrized integrals (to support Triqs; see https://triqs.github.io/triqs/latest/documentation/manual/triqs/operators/util/hamiltonians.html?highlight=hamiltonian#module-triqs.operators.util.hamiltonians_)
 <-> symmetrized (done)

**Comment by PaulWAyers** [comment]
Update: @gabrielasd and I worked through this yesterday, and I think we have a reasonable grasp of how to link to Triqs and what we need to do math-wise.

**Comment by PaulWAyers** [comment]
Good (great) reference on chemists' vs. physicists' notation.
http://vergil.chemistry.gatech.edu/notes/permsymm/permsymm.html

FCIDump and PySCF seem to use chemists' notation.

TL;DR:
In physics notation, the integrand is like <K*(1)L*(2)N(2)M(1)>. For chemists it is like <K*(1)M(1)L*(2)N(2)>.
